contracts/SponsorFaucet.sol

```diff
@@ -49,6 +88,12 @@ contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard {
         gas_bound = gasBound;
         collateral_bound = collateralBound;
         upper_bound = upperBound;
+        require(generalUpperBound.mul(1000) <= generalGasBound, "upperBound too high");
```

Comment:
Is it possible to define a `modifier` for such `require` statement?

I can see many methods have such validations.

Reply:
Since **struct** is used as params, **modifier** maybe complicated then **require**.

Reply:
only two **require** now, two **modifier** looks ugly